src/send/SendAmount/useTransactionCallbacks.ts

```diff
@@ -64,9 +64,9 @@ function useTransactionCallbacks({
       inputAmount: inputIsInLocalCurrency ? localAmount! : tokenAmount,
       tokenAmount,
       amountIsInLocalCurrency: inputIsInLocalCurrency,
-      tokenAddress: transferTokenAddress,
+      tokenAddress: tokenInfo!.address!,
     }),
-    [recipient, tokenAmount, transferTokenAddress]
+    [recipient, tokenAmount, transferTokenId]
```

Comment:
```suggestion
    [recipient, tokenAmount, tokenInfo]
```

Reply:
I think using transferTokenId in places like this is relying on the assumption that tokenInfo will always be static relative to tokenId. If we added something to tokenInfo in the future like price or balance, that would no longer be the case, and we could get bugs (if we also used that dynamic property in the callback).

Reply:
Similar issue to https://github.com/valora-inc/wallet/pull/4242#discussion_r1351131006.